src/components/v2/V2Create/forms/FundingForm/index.tsx

```diff
@@ -134,7 +160,8 @@ export default function FundingForm({ onFinish }: { onFinish: VoidFunction }) {
   }, [fundingForm, fundingCycleData, fundAccessConstraint, payoutGroupedSplits])
 
   const onFundingFormSave = useCallback(
-    (fields: FundingFormFields) => {
+    async (fields: FundingFormFields) => {
+      await fundingForm.validateFields()
```

Comment:
is the form not already validated at this point? Using the `onFinish` prop on the form is meant to validate before the `onFinish` handler (this function) is called: https://github.com/jbx-protocol/juice-interface/pull/845/files#diff-54e2ecc8080eb485aa06e45b065668919de95ef981ee2178a191d5971bb524a7L213

could def be broken though knowing `antd`

Reply:
You are correct. Removed